datafusion/core/src/physical_optimizer/pipeline_checker.rs

```diff
@@ -314,7 +314,7 @@ mod sql_tests {
                   FROM test
                   LIMIT 5".to_string(),
             cases: vec![Arc::new(test1), Arc::new(test2)],
-            error_operator: "Window Error".to_string()
+            error_operator: "Sort Error".to_string()
```

Comment:
lgtm, thanks for taking this, why the error message changed to Sort Error, I dont see the respective logic change..

Reply:
I think this happens because when the task of erroring out is left to `PipelineChecker`, which is the last rule, sorts are already in-place. While checking, the algorithm first encounters the sort (before the window), which is is pipeline-breaking. Hence the error messsage.